ios/FluentUI/Vnext/List/ListTokens.swift

```diff
@@ -58,15 +63,15 @@ class MSFListTokens: MSFTokensBase, ObservableObject {
     }
 }
 
-class MSFListCellTokens: MSFTokensBase, ObservableObject {
-    @Published public var backgroundColor: UIColor!
+class MSFCellBaseTokens: MSFTokensBase, ObservableObject {
```

Comment:
Why do we need the MSFCellBaseTokens class?
Can't we just have it all on the MSFListCellTokens and override what's needed in the MSFPersonaViewTokens?

Reply:
As spoken offline, due to a limitation of SGen that will create the same extension for both child classes (defining the class members), the PersonaView tokens class can't inherit from the List tokens class.